style/global.scss

```diff
@@ -118,3 +118,12 @@ header .site-logo {
     --mw: 960px;
   }
 }
+
+.report-covers .content-covers-block {
+  & &-wrap &-image img {
```

Comment:
```scss
.foo {
  &-bar {
    margin: auto;
  }
}
```

becomes

```css
.foo-bar {
  margin: auto;
}
```

---

```scss
.foo {
  & &-bar {
    margin: auto;
  }
}
```

becomes

```css
.foo .foo-bar {
  margin: auto;
}
```

---

Thus 

```scss
.report-covers .content-covers-block {
  & &-wrap &-image img {
    margin: auto;
  }
}
```

becomes

```css
.report-covers .content-covers-block .report-covers .content-covers-block-wrap .report-covers .content-covers-block-image img {
  margin: auto;
}
```

Reply:
What I've written here (I'm pretty sure) is correct, but I'm *not* sure if it's what we actually want; It's been a while since I've looked at this but it looks kinda ridiculously long.